pom.xml

```diff
@@ -19,6 +19,30 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <version>1.12</version>
+        <executions>
+          <execution>
+            <id>regex-property</id>
+            <goals>
+              <goal>regex-property</goal>
+            </goals>
+            <configuration>
+              <name>maven.test.skip</name>
+              <value>${project.artifactId}</value>
+              <regex>vlingo-symbio-foundationdb</regex>
```

Comment:
Ok, I see now. You are checking out the project but not building. Ok, ignore my previous comment.

Reply:
Unfortunately, this didn't work. I removed the module again.